test/input.txt

```diff
@@ -44,6 +44,8 @@
   list
   add Betsy Choo pp/222222 pe/benchoo@nus.edu.sg pa/222, beta street t/secretive
   list
+  add Aaron Tan p/121212 e/aaron@email.com a/12, aa st t/dummy
+  sort
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.